.github/workflows/build.yml

```diff
@@ -3,8 +3,7 @@ name: ci
 on:
   push:
     branches:
-      - "main"
-      - "development"
+      - "*"
```

Comment:
Please remove
```
    branches:
      - "*"
```
since `on: push` would automatically trigger on every branch anyway, adding `branch: *`  is redundant